Safety/Safety-Systems/GIS.rst

```diff
@@ -1,6 +1,325 @@
+
+.. MUFA: Multiple-Use Fiber Access
+
+
+.. _Safety-Systems-GIS-Safety-Matrix-Detections-and-Actions:
+
+Safety Matrix. Detections and Actions
+=======================================
+
+
+The GIS uses a `safety matrix`_ to manage the interlocks, consisting of **Detections** signals 
+and their corresponding **Actions** for the subsystems.
+
+.. _`safety matrix`: https://github.com/lsst-ts/ts_tma_gis-documentation_user-interface-manual/blob/master/media/media/image50.png
+
+.. admonition:: Detections
+
+    Specific conditions or events within a subsystem.
+
+.. admonition:: Action
+    
+    Response to the corresponding detection.
+
+
+Certain Detections that could compromise the system's safety **trigger an interlock** mechanism, 
+which is an Action taken to ensure safety.
+
+Some Detections have Actions that only indicate their status, e.g. A fire interlock.
+
+.. make the example more clear
+
+.. list-table:: Examples
+ :widths: 20 30
+ :header-rows: 1
+
+ * - Detection
+   - Action
+ * - M1M3 mirror cell lights on 
+        (D-18 in GIS)
+
+   - TMA drives will be **STO (Safety Torque Off)** and engage the brakes.
+ * -
+   -
+ * - Big earthquake 
+        (D-2 in GIS)
+   - This will bring down most of the system. 
+        Engage TMA brakes, discharge capacitor banks, and STO the following components: TMA drives, Camera Cable Wrap drives, M1M3 actuators, M2 hexapods, M2 actuators, Camera Rotator, Camera Hexapod, etc.
+
+
+
+.. The page T&S Safety Interlocks - (Franco Colleoni) has a link to the “GIS Operation User manual”.  Is there a current version?. Will look for it. 
+.. https://confluence.lsstcorp.org/pages/viewpage.action?pageId=16318660
+
+
+.. _Safety-Systems-GIS-Interacting-with-GIS:
+
+Interacting with GIS   
+======================
+
+.. _Safety-Systems-GIS-Main-Controller-Interface:
+
+Main Controller Interface
+------------------------------
+
+.. figure:: ./_static/GIS-Main-Panel-Cabinet.jpeg
+      :width: 250px
+      :align: left
+      
+      GIS Main Panel Cabinet at level 2
+
+The Observing Specialists will interact with the Main Controller cabinet, located on Level 2.
+
+Occasionally, in the event of system errors, interaction with other cabinets, for reboots or power cycling, may be necessary. However, such actions are likely beyond the scope of responsibilities for the OS team.
+
+
+.. figure:: ./_static/GIS-Main-Panel-home.png
+      :width: 700px
+      
+      GIS General Overview
+
+
+The GIS General Overview (home), allows you to see the general status of the safety areas, 
+you can select one in the navigation bar to the left.
+
+Some area names are straightforward, while others are:
+
+* **AcFiEa**: Access Control System / Fire Main Controller / Earthquake Control System. The safety gates to the pier intermediate level and to level 8 are referred as access control system.
+
+* **M2CAM**: M2 area, includes the M2 actuator, the M2 hexapod, the Camera Hexapod and the Camera Rotator.
```

Comment:
What do you mean by M2 actuator?

Reply:
Is the name given to that section, not sure if it has a meaning, I will ask

Reply:
It refers to the actual actuators of M2